src/pages/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Card } from "../../components";
+import { useUserQuery } from "../../hooks";
+import "./Wishlist.css";
+
+const Wishlist = () => {
+  const { data } = useUserQuery();
+
+  return (
+    <main className="wishlist mb-8">
+      <h3 className="h3 ta-center mt-3 mb-6">Wishlist</h3>
+
+      {data?.user.wishlist.length === 0 ? (
+        <div className="ta-center mx-2">
+          No items in your wishlist{" "}
+          <Link to="/products" className="link">
+            go to products
+          </Link>{" "}
```

Comment:
remove formatted space

Reply:
these are necessary spaces, if i remove them `No items in your wishlistgo to productspage and add....` gets rendered